clang/docs/ReleaseNotes.rst

```diff
@@ -96,6 +96,10 @@ C++20 Feature Support
   behavior can use the flag '-Xclang -fno-skip-odr-check-in-gmf'.
   (`#79240 <https://github.com/llvm/llvm-project/issues/79240>`_).
 
+- Initial support for class template argument deduciton (CTAD) for type alias
+  templates (`P1814R0 <https://wg21.link/p1814r0>`_).
```

Comment:
Probably should update to at least a `<td class="partial" align="center">Partial</td>` under
https://github.com/llvm/llvm-project/blob/e946b5a87b2db307da076093d0a9a72ecb4ec089/clang/www/cxx_status.html#L843-L847

Reply:
Done.